pre_commit/repository.py

```diff
@@ -190,6 +191,21 @@ def _cloned_repository_hooks(repo_config, store, root_config):
     )
 
 
+_TOP_LEVEL_KEYS = ('repo', 'rev', 'hooks')
+
+
+def _check_extra_top_level_keys(repo_config):
+    extra_top_level_keys = set(dict(repo_config).keys()) - set(_TOP_LEVEL_KEYS)
+    if extra_top_level_keys:
+        logger.warning(
+            'Unexpected top level keys present: {}, only supported'
```

Comment:
let's use consistent messaging with the warning on hook-dict extra keys (it's in this module under `create`)